lib/Doctrine/Migrations/Tools/Console/Command/DumpSchemaCommand.php

```diff
@@ -144,4 +141,14 @@ public function execute(
 
         return 0;
     }
+
+    private function checkNoPreviousDumpExistsForNamespace(string $namespace) : void
+    {
+        $versions = $this->getDependencyFactory()->getMigrationRepository()->getMigrations();
```

Comment:
Are you sure you want to name that variable `$versions` rather than `$migrations`? I think it's confusing…

Reply:
true. fixed